pypopquiz/backends/moviepy.py

```diff
@@ -16,6 +21,46 @@ def silence(_) -> float:
     return 0
 
 
+def make_sin(time_seconds: np.array) -> List[float]:
+    """Create a single tone."""
+    out = [np.sin(1500 * np.pi * time_seconds)]
+    return out
+
+
+@audio_video_fx
+def tone_in_interval(clip: med.AudioClip, interval: Tuple[float, float], freq_hz: int):
+    """Fx that replaces interval by a beep."""
+
+    def beeped(get_frame: Callable, time_seconds: Union[float, np.array]):
+        """Fx callback that replaces the original sounds with a tone in an interval."""
+        t = time_seconds  # pylint: disable=invalid-name
+        original = get_frame(t)
+
+        if np.isscalar(t):
+            t = typing.cast(float, t)  # pylint: disable=invalid-name
+            if interval[0] < t < interval[1]:
+                t_offset = t - interval[0]
+                out = [np.sin(freq_hz * np.pi * t_offset)]
```

Comment:
You can already return here and in line 45 and 59 as well, then you don't need all the else statements, right? Makes the logic a little bit easier to follow perhaps.